packages/commons/src/auth/authorization.ts

```diff
@@ -2,17 +2,19 @@ import { unauthorizedError } from "pagopa-interop-models";
 import { P, match } from "ts-pattern";
 import { z } from "zod";
 import { AppContext } from "../context/context.js";
-import { NonEmptyArray } from "../index.js";
 import {
+  NonEmptyArray,
+  SystemRole,
+  systemRole,
   UserRole,
+  userRole,
+} from "../index.js";
```

Comment:
```suggestion
} from "../index.js";
import { userRole, systemRole, SystemRole, UserRole } from "./roles.js";
```
Since we are inside the same folder I would use import these directly from their source instead from `index`

Reply:
This is really interesting point, that wrong import causes runtime error! 
While searching for the usage of imports, I found several instances so have tracked them in Jira [here](https://pagopa.atlassian.net/browse/PIN-6826?atlOrigin=eyJpIjoiOGM3NzViYWY2MmEwNGZhZGE5MTg2ZjcwMjUzOTI3YTAiLCJwIjoiaiJ9).